usb.go

```diff
@@ -168,6 +168,23 @@ func NewContext() *Context {
 	return newContextWithImpl(libusbImpl{})
```

Comment:
would it make sense to replace this with "return ContextOptions{}.New()"? It's now becoming essentially a "context with defaults" anyway.

Reply:
I agree - it also matches better the 'new' libusb initializer, since the int `libusb_init(libusb_context  ctx)` is deprecated in favour of always calling it with options.